modules/installation-configuration-parameters.adoc

```diff
@@ -1,54 +1,110 @@
+cluster's platform.  During the initialization phase of installation, you
+provide values for the required parameters through the command line. If you
+customize your cluster, you can modify the `./install-config.yaml` file to
+provide more details about the platform.
 
 [NOTE]
 ====
 You cannot modify these parameters after installation.
 ====
 
-.Parameters that you define during initialization
-[cols=".^2,.^3,.^3a",options="header"]
+.Required parameters
+[cols=".^2,.^3,.^5a",options="header"]
 |====
 |Parameter|Description|Values
 
 |`provider`
 |The cloud provider to host the cluster.
-a|Enter one of the following values:
+|`aws`
+
+|`baseDomain`
+|The base domain of your cloud provider. This value is used to create routes
+to your {product-title} cluster components.
+|A fully-qualified domain or subdomain name, such as `dev.example.com`.
+
+|`name`
+|The name of your cluster.
+|A string that contains uppercase or lowercase letters, such as `dev`.
+
+|`region`
+|The name of your cluster.
+|A valid AWS region, such as `us-east-1`.
 
-* aws
-* azure
-* openstack
+|`pullSecret`
+|The pull secret that you obtained from link:try.openshift.com[the OpenShift start page].
+|
+[source,json]
+----
+{  
+   "auths":{  
+      "cloud.openshift.com":{  
+         "auth":"b3Blb=",
+         "email":"you@example.com"
+      },
+      "quay.io":{  
+         "auth":"b3Blb=",
+         "email":"you@example.com"
+      }
+   }
+}
+----
 
-|`base-domain`
-|The base domain of your cloud provider. This value is used to <do a thing>.
-|A FQDN, such as `dev.example.com`.
+|`sshKey`
+|The SSH key to use to access your cluster machines.
+|A valid public SSH key.
 |====
 
-.All installation parameters
-[cols=".^3,.^5,.^5a,.^5",options="header"]
+
+.Optional AWS platform parameters
+[cols=".^2,.^3,.^3a",options="header"]
 |====
-|Parameter|Description|Value format|Default value
+|Parameter|Description|Values
 
-|`provider`
-|The cloud provider to host the cluster.
-a|Enter one of the following values:
+|`machines.platform.aws.iamRoleName`
+|The IAM role assigned to the machines in the pool.
+|
+
+|`machines.platform.aws.rootVolume.iops`
+|The Input/Output Operations Per Second (IOPS) that is reserved for the root volume.
+|Integer, for example `4000`
+
+|`machines.platform.aws.rootVolume.size`
+|The size in GiB of the root volume.
+|Integer, for example `500`
+
+|`machines.platform.aws.rootVolume.type`
+|The instance type of the root volume.
+|Valid link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html[AWS EBS instance type],
+such as `io1`.
+
+|`machines.platform.aws.type`
+|The EC2 instance type for the control plane machines.
+|Valid link:https://aws.amazon.com/ec2/instance-types/c5/[AWS instance type],
+such as `c5.9xlarge`.
+
+|`machines.platform.aws.zones`
+|The availability zones where the installation program creates machines for this
+pool.
+|Valid AWS availability zone, such as `us-east-1c`.
```

Comment:
Mention that this is [a YAML sequence][1]?  Although you could probably use "list" or "array" as synonyms, since YAML doesn't distinguish at that level of detail, and the other terms may be more familiar for readers, I dunno.  [JSON uses "array"][2].  Makes it nice and easy, right? :p

[1]: https://yaml.org/spec/1.2/spec.html#sequence//
[2]: https://tools.ietf.org/html/rfc7159#section-5

Reply:
Why not both? I'll call it a list and provide that yaml sequence link.

Reply:
> Why not both? I'll call it a list and provide that yaml sequence link.

Fine with me.